app/src/main/res/values-it/strings.xml

```diff
@@ -26,16 +33,14 @@
     <string name="gotorating">Portami!</string>
     <string name="nevershowagain">Lasciami stare!</string>
     <string name="reconsider">Forse dopo</string>
-    <string name="donation_message">Sembra che Substratum Extras for Android Auto ti piaccia! Vorresti aiutarmi facendo una piccola donazione?</string>
-    <string name="helpButton">🆘 Aiuto 🆘</string>
+    <string name="donation_message">Sembra che Substratum Extras for Android Auto ti piaccia!\nVorresti aiutarmi facendo una piccola donazione?</string>
```

Comment:
No need for a new line.

Reply:
The strings with multi period with on same line without a forced CR/LF are less readable than string that have liens for each epriod.